src/matrix/inversewishart.jl

```diff
@@ -50,11 +50,11 @@ params(d::InverseWishart) = (d.df, d.Ψ, d.c0)
 
 ### Conversion
 function convert(::Type{InverseWishart{T}}, d::InverseWishart) where T<:Real
-    P = Wishart{T}(d.Ψ)
+    P = convert(AbstractArray{T}, d.Ψ)
```

Comment:
The conversion changes made here in `src/matrix/wishart.jl` make sense to me, but these do not. Shouldn't `P` still be a `Wishart` rather than an array?

Reply:
I don't think so? The `Ψ` field of `InverseWishart` is a `PDMat`.

I don't understand what `P = Wishart{T}(d.Ψ)` was ever supposed to mean.

Reply:
Fair point. Thanks for explaining.